Makefile

```diff
@@ -68,13 +90,21 @@ builder:
 # install qemu emulators
 	docker run -it --rm --privileged tonistiigi/binfmt --install all
 
-
 $(LIBGIT2): $(MUSL-CC)
 ifeq ($(shell uname -s),Darwin)
+ifeq ($(LIBGIT2_ONLY),true)
+	TARGET_DIR=$(TARGET_DIR) BUILD_ROOT_DIR=$(BUILD_ROOT_DIR) \
+		./hack/static.sh build_libgit2_only
+else
 	TARGET_DIR=$(TARGET_DIR) BUILD_ROOT_DIR=$(BUILD_ROOT_DIR) \
 		./hack/static.sh all
+endif
 else
+ifeq ($(LIBGIT2_ONLY),true)
 	IMG_TAG=$(IMG):$(TAG) ./hack/extract-libraries.sh
+else
+	IMG_TAG=$(IMG):$(TAG) ./hack/extract-libraries.sh
+endif
```

Comment:
`Ifeq` seems redundant here, not sure we need it here.

Reply:
Yep, it is, sorry about that, seems to have slipped through some outdated refactoring.